#! /bin/sh /usr/share/dpatch/dpatch-run
## mono_transition_use_csc.dpatch by Iain Lane <laney@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch build system to use compiler from MCS variable instead of hardcoded gmcs. Part of mono 2.0 transition.

@DPATCH@
diff -urNad f-spot-0.5.0.3~/Makefile.include f-spot-0.5.0.3/Makefile.include
--- f-spot-0.5.0.3~/Makefile.include	2008-07-08 12:01:42.000000000 +0100
+++ f-spot-0.5.0.3/Makefile.include	2008-11-30 20:27:01.000000000 +0000
@@ -40,8 +40,8 @@
 ## Build
 
 CSC_FLAGS = -debug
-CSC = gmcs $(CSC_FLAGS) -target:exe
-CSC_LIB = gmcs $(CSC_FLAGS) -target:library
+CSC = $(MCS) $(CSC_FLAGS) -target:exe
+CSC_LIB = $(MCS) $(CSC_FLAGS) -target:library
 
 ## Runtime
 
diff -urNad f-spot-0.5.0.3~/configure.in f-spot-0.5.0.3/configure.in
--- f-spot-0.5.0.3~/configure.in	2008-10-17 19:54:15.000000000 +0100
+++ f-spot-0.5.0.3/configure.in	2008-11-30 20:26:23.000000000 +0000
@@ -315,7 +315,7 @@
 AC_SUBST(CPPFLAGS)
 AC_SUBST(LDFLAGS)
 
-CSC=gmcs
+CSC=$(MCS)
 AC_SUBST(CSC)
 
 expanded_libdir=`( case $prefix in NONE) prefix=$ac_default_prefix ;; *) ;; esac
